scripts/dtlscid.test

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+set -e
+set -x
+
+cleanup () {
+    echo
+    echo "Cleaning up..."
+    if [ ! -z "$UDP_PROXY_PID" ];then
+        echo "Killing udp_proxy $UDP_PROXY_PID"
+        kill $UDP_PROXY_PID
+    fi
+    if [ ! -z "$SERVER_PID" ];then
+        echo "Killing server $SERVER_PID"
+        kill $SERVER_PID
+    fi
+}
+
+trap cleanup err exit
+
+CCID="AA"
+SCID="BB"
+WOLFSSL_ROOT=$(pwd)
+SERVER_PORT=11111
+
+(killall udp_proxy || true)
+(killall lt-server || true)
+(killall lt-client || true)
+
+test_cid () {
+    $WOLFSSL_ROOT/examples/server/server -v4 -u --cid $SCID 2>/tmp/serr 1>/tmp/sout &
+    SERVER_PID=$!
+    sleep 0.2
```

Comment:
This script does pretty much the same thing that `test-dtls13-cid.conf` does. I think it would be better to expand this script to test a multiplexing server (once implemented).

Reply:
I agree with you, but I would do this in a separate PR as it's quite unrelated to CID and it will touch dtls13.test (and maybe tls13.test) as well.